packages/framework/resources/views/layouts/publication_list.blade.php

```diff
@@ -4,13 +4,28 @@
         <div class="prose dark:prose-invert">
             @php/** @var \Hyde\Framework\Features\Publications\Models\PublicationType $publicationType*/@endphp
             <h1>Publications for type {{ $publicationType->name }}</h1>
-            <ol>
-                @foreach($publicationType->getPublications() as $publication)
-                    <li>
-                        <x-link :href="$publication->getRoute()">{{ $publication->title }}</x-link>
-                    </li>
-                @endforeach
-            </ol>
+            @if($publicationType->usesPagination())
```

Comment:
Probably want to extract this to a separate view to keep it light

Reply:
Thinking a nice flow could be when creating a publication type, instead of asking if the user wants to configure pagination, we ask if the want to enable it. If enabled we present the questions (with defaults that can just be entered by hitting enter), and then we copy over the paginated template, if not enabled, we just use the default one.